docs/topics/cache.txt

```diff
@@ -293,6 +293,11 @@ above example, if your server runs as the user ``apache``, make sure the
 directory ``/var/tmp/django_cache`` exists and is readable and writable by the
 user ``apache``.
 
+.. warning::
+    **If the caches LOCATION is a sub directory of** :setting:`MEDIA_ROOT`, :setting:`STATIC_ROOT` **or** :setting:`STATICFILES_FINDERS`, **this can lead to arbitrary remote code execution or information leaking.**
+
+    An attacker access the cache file can not only falsified html content, which your site will trust, but also remotely execute arbitrary code, as the data is serialized using pickle.
```

Comment:
"An attacker who gains access to the cache file can not only falsify"

Reply:
Thanks a lot, sorry about my poor English!

Reply:
no problem!